packages/openid4vci/src/formats/credential/mso-mdoc/z-mso-mdoc.ts

```diff
@@ -3,12 +3,16 @@ import {
   zCredentialConfigurationSupportedClaimsDraft14,
   zMsoMdocIssuerMetadataClaimsDescription,
 } from '../../../metadata/credential-issuer/z-claims-description'
-import { zCredentialConfigurationSupportedCommonCredentialMetadata } from '../../../metadata/credential-issuer/z-credential-configuration-supported-common'
+import {
+  zCredentialConfigurationSupportedCommon,
+  zCredentialConfigurationSupportedCommonCredentialMetadata,
+  zCredentialConfigurationSupportedCommonDraft15,
+} from '../../../metadata/credential-issuer/z-credential-configuration-supported-common'
 
 export const zMsoMdocFormatIdentifier = z.literal('mso_mdoc')
 export type MsoMdocFormatIdentifier = z.infer<typeof zMsoMdocFormatIdentifier>
 
-export const zMsoMdocCredentialIssuerMetadata = z.object({
+export const zMsoMdocCredentialIssuerMetadata = zCredentialConfigurationSupportedCommon.extend({
```

Comment:
This does change the current behaviour, because we filtered on the credential format specfic params, and now it's also the common params. But I think if everything works fine with this it's fine, just to be careful since it was only the specific params in the beginning

Reply:
Yes, indeed. I hadn't noticed we were exporting this to the user of the package, I thought it was just internal. This was needed to solve some issues with the typings unfortunately.